tests/server_test.py

```diff
@@ -80,13 +80,23 @@ def test_elegant_data_file(fc):
 
 def test_myapp_basic(fc):
     from pykern import pkunit, pkcompat
+    from pykern.pkunit import pkok
 
     r = fc.get('/old')
     assert 'LandingPageController' in pkcompat.from_bytes(r.data), \
         'Top level document is the landing page'
     r = fc.get('/robots.txt')
     pkunit.pkre('elegant.*myapp.*srw', pkcompat.from_bytes(r.data))
 
+    r = fc.get('/en/landing.html')
+    pkok(
+        not re.search(
+            r'https://www.googletagmanager.com/gtm.js\?id=',
+            pkcompat.from_bytes(r.data)
+        ),
+        'Unexpected injection of google tag mgr',
```

Comment:
dump the data for debugging: `'Unexpected injection of googletag data={}', r.data`